### ISSUE TITLE
Request for Simblee nrf51822 board support

### ISSUE THREAD

**Issue by danlsk** [platformio/platform-nordicnrf51#6]
Simblee Arduino library : [Board Manager URL](https://www.simblee.com/package_simblee166_index.json).
The library is not in github but you can get the files after installation in Arduino.

**Comment by ivankravets** [comment]
Would you like to help us and provide support? It should be easy.
Here is docs http://docs.platformio.org/en/stable/platforms/creating_platform.html

**Comment by danlsk** [comment]
Sure, I can try. In return, I get free upgrade? Joking.....

**Comment by ivankravets** [comment]
> In return, I get free upgrade? Joking.....

Yes, you will receive free upgrade from PlatformIO 3.1 to PlatformIO 3.2 😂

**Comment by simonem** [comment]
@danlsk did you manage to use simblee in platformio?

**Comment by danlsk** [comment]
Nope, I didn't tried. I stop using PlatformIO at the moment.
